Makefile

```diff
@@ -94,7 +97,6 @@ define run-in-test-server
 		-v ${GOPATH}/pkg/mod:/go/pkg/mod${VOLUME_MOUNT} \
 		-v ${GOCACHE}:/tmp/go-build-cache${VOLUME_MOUNT} \
 		-v ${HOME}/.kube:/home/user/.kube${VOLUME_MOUNT} \
-		-v /tmp:/tmp${VOLUME_MOUNT} \
```

Comment:
I’m not sure why `/tmp` was bind-mounted (for caching reasons maybe?), but one cannot relabel (change SELinux label, which happens when using `:Z` on a mount) `/tmp`.